resources/run_adl.sh

```diff
@@ -0,0 +1,195 @@
+    mkdir -p $TMP_DIR
+
+    # Note: ADL has a storage.json file that generates configs for us.
+    # The mongodb source is on port $MONGOD_PORT so we use that here.
+    ${MONGO_DOWNLOAD_LINK:0:$((${#MONGO_DOWNLOAD_LINK} - 4))}/bin/mongod --port $MONGOD_PORT --dbpath $MONGO_DB_PATH \
+      --logpath $LOGS_PATH/mongodb_test.log --pidfilepath $TMP_DIR/${MONGOD}.pid --fork
+  fi
+else
+  if [ $ARG = $STOP ]; then
+    MONGOD_PID=$(< $TMP_DIR/${MONGOD}.pid)
+    echo "Stopping $MONGOD, pid $MONGOD_PID"
+    kill "$(< $TMP_DIR/${MONGOD}.pid)"
+  fi
+fi
+
+check_mongohoused
+if [[ $? -ne 0 ]]; then
+  if [ $ARG = $START ]; then
+    # Install and start mongohoused
+    git config --global url.git@github.com:.insteadOf https://github.com/
+    # Clone the mongohouse repo
+    if [ ! -d "$MONGOHOUSE_DIR" ]; then
+      git clone $MONGOHOUSE_URI
+    fi
+    cd $MONGOHOUSE_DIR
+    git pull $MONGOHOUSE_URI
+
+    go mod download
+
+    # Download external dependencies
+    go run cmd/buildscript/build.go tools:download:mqlrun
+    go run cmd/buildscript/build.go tools:download:mongosql
+
+    # Set relevant environment variables
+    export GOPRIVATE=github.com/10gen
+    export MONGOHOUSE_ENVIRONMENT="local"
+    export MONGOHOUSE_MQLRUN="$(pwd)/artifacts/mqlrun"
+    export LIBRARY_PATH="$(pwd)/artifacts"
+
+    # Load tenant config into mongodb
+    STORES='{ "name" : "localmongo", "provider" : "mongodb", "uri" : "mongodb://localhost:%s" }'
+    STORES=$(printf "$STORES" "${MONGOD_PORT}")
+    DATABASES='{ "name" : "integration_test", "collections" : [ { "name": "*", "dataSources" : [
+      { "storeName" : "localmongo", "database" : "integration_test" } ] } ], "views" : null }'
+    TENANT_CONFIG="./testdata/config/mongodb_local/tenant-config.json"
+    # Replace the existing storage config with a wildcard collection for the local mongodb
+    jq "del(.storage)" ${TENANT_CONFIG} > ${TENANT_CONFIG}.tmp && mv ${TENANT_CONFIG}.tmp ${TENANT_CONFIG}
+    jq --argjson obj "$STORES" '.storage.stores += [$obj]' ${TENANT_CONFIG} > ${TENANT_CONFIG}.tmp\
+                                                               && mv ${TENANT_CONFIG}.tmp ${TENANT_CONFIG}
+    jq --argjson obj "$DATABASES" '.storage.databases += [$obj]' ${TENANT_CONFIG} > ${TENANT_CONFIG}.tmp\
+                                                               && mv ${TENANT_CONFIG}.tmp ${TENANT_CONFIG}
+
+    go run cmd/buildscript/build.go init:mongodb-tenant
+
+    mkdir -p $TMP_DIR
+    mkdir -p $LOGS_PATH
+    # Start mongohoused with appropriate config
+    nohup go run -tags mongosql ./cmd/mongohoused/mongohoused.go \
+      --config ./testdata/config/mongodb_local/frontend-agent-backend.yaml >> $LOGS_PATH/${MONGOHOUSED}.log &
+    echo $! > $TMP_DIR/${MONGOHOUSED}.pid
+  fi
+else
+  if [ $ARG = $STOP ]; then
+    MONGOHOUSED_PID=$(< $TMP_DIR/${MONGOHOUSED}.pid)
+    echo "Stopping $MONGOHOUSED, pid $MONGOHOUSED_PID"
+    pkill -TERM -P ${MONGOHOUSED_PID}
+  fi
+fi
```

Comment:
```suggestion
fi

```
[nit] ending with newline